improver_tests/utilities/solar/test_DayNightMask.py

```diff
@@ -92,7 +97,12 @@ class Test__daynight_lat_lon_cube(IrisTest):
 
     def setUp(self):
         """Set up the cube for testing."""
-        cube = set_up_cube_lat_long()
+        data = np.ones((16, 16), dtype=np.float32)
+        cube = set_up_variable_cube(
+            data,
+            "precipitation_amount",
+            "kg m^-2",
+        )
         lon_points = np.linspace(-8, 7, 16)
         lon_points_360 = np.linspace(345, 360, 16)
         lat_points = np.linspace(49, 64, 16)
```

Comment:
New arguments to `set_up_variable_cube` will allow you to set up the required latitude and longitude points from the start.  Have a look at the `grid_spacing` and `domain_corner` arguments added by @VictoriaLouiseS

Reply:
Done.